packages/template-block/src/settings.ts

```diff
@@ -74,6 +76,44 @@ export const settings = defineSettings({
             show: (bundle: Bundle) => bundle.getBlock('preview')?.value === PreviewType.Custom,
             showForTranslations: true,
         },
+        {
+            id: 'templateEditingHeading',
+            type: 'sectionHeading',
+            blocks: [
+                {
+                    id: 'templateEditing',
+                    label: 'Template Editing',
+                    info: 'Choose how users can make changes to the template.',
+                    type: 'slider',
+                    choices: [
+                        { label: 'Simple', value: TemplateEditing.Simple },
+                        { label: 'Advanced', value: TemplateEditing.Advanced },
+                    ],
+                    defaultValue: TemplateEditing.Simple,
+                    showForTranslations: true,
+                },
+                {
+                    id: 'templateEditingNotificationSimple',
+                    type: 'notification',
+                    text: 'Users can only make changes by selecting and editing fields in the settings panel without approvals. Read <article name> to learn more.',
```

Comment:
Is it even possible to have a link here in place of `<articleName>`?

Reply:
But generally, I would think better to leave out the `Read <article name> to learn more.` entirely until we have an actual article to put there

Reply:
Would be possible to use a link by moving this to a `footer` section of the notifications element I believe.  But we can revisit this once we have copy